src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -248,6 +248,98 @@ export default defineConfig({
 });
 ```
 
+### `workerEntryPoint`
+<p>
+
+**Type:** `{path: string | URL, exports: string[]}`
+**Default:** `{ path: '@astrojs/cloudflare/entrypoints/server.js', exports: ['default'] }`
```

Comment:
```suggestion
**Default:** `{ path: '@astrojs/cloudflare/entrypoints/server.js', namedExports: [] }`
```
@alexanderniebuhr this is what I wanted to check on! Is this just not updated for the new decision, or is `exports` here an internal (non-directly exposed) property that combines default and named exports? It seems weird to show that here instead of `namedExports`, which is what *is* exposed and configurable to the user, hiding the notion of default exports at all.